This package no longer builds in release mode on linux

Building in release mode produces this since #410 was merged:
```
> ./swift-5.6.2-RELEASE-ubuntu20.04/usr/bin/swift build -c release
/home/butta/swift-argument-parser/Sources/ArgumentParserTestHelpers/TestHelpers.swift:12:18: error: module 'ArgumentParser' was not compiled for testing
@testable import ArgumentParser
```
That means the last working version was 1.0.3. A workaround is to add `-Xswiftc -enable-testing`.

**ArgumentParser version:** latest commit df9ee66 from the  `main` branch
**Swift version:** 5.5 and later, probably earlier versions too

### Checklist
- [x] If possible, I've reproduced the issue using the `main` branch of this package
- [x] I've searched for [existing GitHub issues](https://github.com/apple/swift-argument-parser/issues)

--- a/Sources/ArgumentParser/Parsable Properties/ArgumentVisibility.swift
+++ b/Sources/ArgumentParser/Parsable Properties/ArgumentVisibility.swift
@@ -10,7 +10,7 @@
 //===----------------------------------------------------------------------===//
 
 /// Visibility level of an argument's help.
-public struct ArgumentVisibility {
+public struct ArgumentVisibility: Hashable {
   /// Internal implementation of `ArgumentVisibility` to allow for easier API
   /// evolution.
   internal enum Representation {

--- a/Sources/ArgumentParserTestHelpers/TestHelpers.swift
+++ b/Sources/ArgumentParserTestHelpers/TestHelpers.swift
@@ -9,7 +9,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-@testable import ArgumentParser
+import ArgumentParser
 import ArgumentParserToolInfo
 import XCTest
 
@@ -122,7 +122,7 @@ public func AssertHelp<T: ParsableArguments>(
   let flag: String
   let includeHidden: Bool
 
-  switch visibility.base {
+  switch visibility {
   case .default:
     flag = "--help"
     includeHidden = false
@@ -130,7 +130,10 @@ public func AssertHelp<T: ParsableArguments>(
     flag = "--help-hidden"
     includeHidden = true
   case .private:
-    XCTFail("Should not be called.")
+    XCTFail("Should not be called.", file: file, line: line)
+    return
+  default:
+    XCTFail("Unrecognized visibility.", file: file, line: line)
     return
   }
 
@@ -158,13 +161,16 @@ public func AssertHelp<T: ParsableCommand, U: ParsableCommand>(
 ) {
   let includeHidden: Bool
 
-  switch visibility.base {
+  switch visibility {
   case .default:
     includeHidden = false
   case .hidden:
     includeHidden = true
   case .private:
-    XCTFail("Should not be called.")
+    XCTFail("Should not be called.", file: file, line: line)
+    return
+  default:
+    XCTFail("Unrecognized visibility.", file: file, line: line)
     return
   }
 

